server/models/Transaction.js

```diff
@@ -86,6 +86,22 @@ module.exports = function(Sequelize, DataTypes) {
   }, {
     paranoid: true,
 
+    classMethods: {
+      createMany: (transactions, defaultValues) => {
```

Comment:
This really should be a Promise.map.

`return Promise.map(transactions, transaction => Transaction.create(_.merge(...)))`